api/candlepin-api-spec.yaml

```diff
@@ -4202,6 +4397,8 @@ components:
 
     GuestIdDTO:
       description: Represents a guest ID running on a virt host consumer
+      required:
```

Comment:
Same here, `required:` needs to be at the same level as `properties:` is

Reply:
Corrected.